docs(storybook): docs cleanup (#6495)
adobe/spectrum-web-components: 2nd-gen/packages/swc/.storybook/guides/customization/component-styles.mdx

Reviewer: ```suggestion
It is strongly encouraged to re-assign color values to existing Spectrum color tokens, which have the built-in capability to adapt to light and dark theme contexts.
```

Reviewer: ```suggestion
If you notice an unexpected visual impact from your override, re-assess how the selector is scoped. For example, you may have used the simple component selector instead of a compound attribute selector.
```

Reviewer: ```suggestion
To exclude a style further, overrides can be applied to specific component instances using custom classes.
```

import { Meta } from '@storybook/addon-docs/blocks';

<Meta title="Customization/Component styles" />

# Component styles

Component styles can be customized in a few ways, listed in the recommended order of implementation:

> 1. Use of style-related attributes per component such as `variant` and `size`

Reviewer: ```suggestion
1. Use style-related attributes per component, such as `variant` and `size`
```

Reviewer: Using "simple" verb tenses instead of -ing ending

> 2. Providing [overrides of component custom properties](#overriding-component-custom-properties)

Reviewer: ```suggestion
2. Provide [overrides of component custom properties](#overriding-component-custom-properties)
```

> 3. _Coming soon:_ writing styles for specific component shadow parts

Reviewer: ```suggestion
3. _Coming soon:_ Write styles for specific component shadow parts
```

> 4. _Coming soon:_ providing granular customization via the theme object

Reviewer: ```suggestion
4. _Coming soon:_ Provide granular customization via the theme object
```

> 5. Extending the components to include custom styles

Reviewer: ```suggestion
5. Extend components to include custom styles
```

> 6. [Globally overriding design token](#globally-override-design-tokens) custom properties

Reviewer: ```suggestion
6. [Globally override design token](#globally-override-design-tokens) custom properties
```

> SWC strongly recommends using the least intrusive method possible for assigning custom styles. This will help keep your component styles overall inline with Spectrum design recommendations, and reduce the likelihood of compatibility issues as the library evolves.

Reviewer: ```suggestion
SWC strongly recommends using the least intrusive method possible for assigning custom styles. This will help keep your component styles in line with Spectrum design recommendations, and reduce the likelihood of compatibility issues as the library evolves.
```

There are some exclusions to what can be successfully overridden:

- Color properties for static white and static black variants
- Color properties when tied to non-semantic color palette variants
  - **Example**: the `magenta` variant of Badge
- Properties modified for forced-colors mode

> To modify non-semantic color palette variants, you may add a global override to re-assign those values instead.

Reviewer: ```suggestion
To modify non-semantic color palette variants, add a global override to re-assign those values instead.
```
